internal/cmd/agent.go

```diff
@@ -48,10 +50,20 @@ coder agent start
 coder agent start --coder-url https://my-coder.com --token xxxx-xxxx
 `,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			var (
-				ctx = cmd.Context()
-				log = slog.Make(sloghuman.Sink(os.Stderr)).Leveled(slog.LevelDebug)
-			)
+			ctx := cmd.Context()
+			sinks := []slog.Sink{
+				sloghuman.Sink(os.Stderr),
+			}
+
+			file, err := os.OpenFile(filepath.Join(os.TempDir(), "coder-agent.log"), os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0600)
```

Comment:
it didn't seem like the file we write needs to be hidden, so I made it a not-dotfile

Reply:
Should be consistent for the other log files in that case

Reply:
Which other log files do we have? I think just a code-server one right?

Reply:
code-server and envagent, they're in environment_stages.go